### ISSUE TITLE
An automatic hidding of empty reference table sactions or inserting a row saying: "This section is empty"

### ISSUE THREAD

**Issue by MichalMaler** [quarkusio/quarkus#43933]
### Description

Hello. Based on QE feedback, I received a suggestion to comment out an empty section mentioned below that contains no data. 
https://quarkus.io/guides/datasource#reactive-oracle-specific-configuration
![Image](https://github.com/user-attachments/assets/996fad05-bf04-4f08-98eb-0cc58784d784)

I asked @yrodiere for his opinion, and he was for keeping it there but creating a ticket that would ask for an implementation of some automatic text line insertion, which would state that this section is currently empty (and it is not empty because of an error). Another way could be to hide the section automatically if it contains no data.

I think such a solution would be great because removing or hiding these empty sections could lead to a situation in which we forget to put them back when they are filled with information in the future.

**Comment by gsmet** [comment]
Hmmm, it was supposed to be handled properly and have a line saying that there were no configuration properties. That's something I fixed with the new config doc actually.

I decided to keep it as we want the properties to appear automatically if we add some.

It worked for a while, something must have changed in the config, I'll have a look.

**Comment by gsmet** [comment]
It's supposed to be handled but there's definitely something fishy, I'll have a closer look later.

**Comment by gsmet** [comment]
I remembered well, I actually took care of it but as part of this 3.16 work: https://github.com/quarkusio/quarkus/pull/42907 .

It's relatively old work but we branched 3.14 a while ago.

I won't backport it to 3.15 as it's part of some more involved work that is definitely not backportable.

It's not a regression as it was already the same in 3.8: https://quarkus.io/version/3.8/guides/reactive-sql-clients#oracle .

So, in summary, will be fixed in the community doc with 3.16.
